source/_posts/2024-12-01-the-month-of-what-the-heck.markdown

```
@@ -0,0 +1,211 @@
+---
+title: The month of 'What the Heck?!' 2024
+description: "Ever felt that 'What the heck Home Assistant?!' moment? This month, we would like to learn about your 'what the heck?!' moments..."
+date: 2024-11-30 00:00:00
+date_formatted: "November 30, 2024"
+author: Franck Nijhof
+author_twitter: frenck
+categories: Announcements
+og_image: /images/blog/2024-12-01-the-month-of-what-the-heck/social.jpg
+---
+
+<a href='https://community.home-assistant.io/c/month-of-what-the-heck/61'><img src='/images/blog/2024-12-01-the-month-of-what-the-heck/social.jpg' style='border: 0;box-shadow: none;'></a>
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)